Unix/http/http.c

```diff
@@ -1477,7 +1477,11 @@ static MI_Result _CreateSSLContext(Http* self, const char* sslCipherSuite, Serve
     {
         options |= SSL_OP_NO_SSLv3;
     }
-    if ( SSL_CTX_set_options(sslContext, options) == 0 )
+#if defined(linux)
```

Comment:
ifdef removed. commit amended. 
New comment written, including a note that perhaps we should error out if we don't get the options we requested (ie if the return value anded with options != options).

I didn't just do that because I don't know if that is the expected behavior for the surrounding function.